docs/configuration-reference/components/cert-manager.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Cert-Manager configuration reference for Lokomotive
+linkTitle: Cert-Manager
```

Comment:
While we're on it, maybe we can change the name to `cert-manager`? This is official name of the project as far as I know.

Reply:
Sure, done.